packages/loader/container-utils/src/error.ts

```diff
@@ -178,7 +178,7 @@ export class DataProcessingError extends LoggingError implements IErrorBase, IFl
 }
 
 export const extractSafePropertiesFromMessage = (message: ISequencedDocumentMessage) => ({
-	messageClientId: message.clientId,
+	messageClientId: message.clientId ?? "",
```

Comment:
Same here - I'd leave the exact clientId used unless/until the server sends something different.

Reply:
that would be a breaking change to support. trying to see if i can keep this non-breaking, as it worked until now will nulls coming from the server

Reply:
I'm not sure I understand - you mean that this function only worked before because it secretly had a guarantee that it was never called with a server-generated message?

Maybe consider string `"null"` then at least?  These sneaky tweaks to telemetry feel like they always cause confusion though (makes me think about when we were lying about online status in error telemetry).

Reply:
no secret guarantee. it just works with null values at runtime, even though it is a violation of the typing

Reply:
```suggestion
	messageClientId: message.clientId === null ? "null" : message.clientId,
```

Reply:
we see active telemetry logging with `null` here even though the type system doesn't know it, so is a bug waiting to happen.